<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>1416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>725</width>
         <height>1327</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>631</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>1. 폴더 생성</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>폴더 이름:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>경로 : </string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>선택</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>생성/지정</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="3">
              <widget class="QLabel" name="label">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>2. 엑셀 파일 업로드</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="2">
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>업로드</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>업로드</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>상품명/발주처 파일 : </string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>주문 파일:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_4"/>
             </item>
             <item row="4" column="0" colspan="3">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="3">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>엑셀 분류 시작</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>3. 이메일 발송(Google만 가능)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_6"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>제목 : </string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pushButton_6">
               <property name="text">
                <string>내용 확인</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QTextEdit" name="textEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>본문 : </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>*구글 비밀번호는 2단계 인증 활성-&gt;앱비밀번호 설정 후 입력해주세요.</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>ID : </string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>PW : </string>
               </property>
              </widget>
             </item>
             <item row="0" column="2" rowspan="2">
              <widget class="QPushButton" name="pushButton_8">
               <property name="text">
                <string>발송하기</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_8">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_9">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>&lt;a href=&quot;https://myaccount.google.com/apppasswords?pli=1&amp;rapt=AEjHL4MGn1uuY2b9MN59aT37kz-EQutjVXglLg6QrIU1_hbUUY5kx-AFjnh65AY8TuxSwhH5-OHIrL3dBwhG03wB4liVZ7tc9idroqMy3C1uYhGNw90g0kU&quot;&gt;구글 앱비밀번호 설정하기&lt;/a&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>
&lt;a href=&quot;https://myaccount.google.com/security&quot;&gt;2단계 인증 활성화하기&lt;/a&gt;
</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>셜록 주문 시스템</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
